src/Services/GlobalStyle.js

```diff
@@ -7,11 +7,22 @@ export default createGlobalStyle`
     padding: 0;
     margin: 0;
     font-size: 16px;
-    font-family: sans-serif;
+    font-family: Silom;
+    font-weight: regular;
+    color: white;
```

Comment:
you could add global variables if you are using the same colors multiple times

Reply:
Thanks for the idea! :)